bank2ynab/bank_process.py

```diff
@@ -77,7 +77,7 @@ def get_files(self):
                 files = [
                     join(path, f)
                     for f in directory_list
-                    if f.endswith(ext)
+                    if f.endswith(ext) or f.endswith(ext.lower()) or f.endswith(ext.upper())
```

Comment:
`if f.lower().endswith(ext) `
as a replacement for the whole of line 80 here should work in theory. It converts the entire filename to lower case and then runs the comparison against that.